README.md

```diff
@@ -2,8 +2,70 @@ This project is a component of the [Operator Framework](https://github.com/opera
 
 # Helm App Operator Kit
 
-This repository serves as a template for easily creating managed stateless applications that run either Kubernetes Deployments or Helm charts. It builds upon the [Lostromos project](https://github.com/wpengine/lostromos).
+This repository serves as a template for easily creating managed stateless applications that run either Kubernetes Deployments or Helm charts. It was inspired by the [Lostromos project](https://github.com/wpengine/lostromos). The underlying Operator was created using the `operator-sdk new` command.
 
-## Getting Started
+# Installing a custom Helm-based app
 
-See the documentation on [Creating a Helm-based App](/Documentation/install-helm-app.md).
+While the [Operator Lifecycle Manager][olm-repo] can only manage Operators, not all applications require developers to write a custom Operator.
+The [Helm App Operator Kit][helm-sdk] makes it possible to leverage a pre-existing Helm chart to deploy Kubernetes resources as a unified application.
+
+```sh
+git clone https://github.com/coreos/helm-app-operator-kit
+cd helm-app-operator-kit
+```
+
+To create a new application, the Helm App Operator Kit provides both a shell script and manual instructions.
+
+## Prerequisites
+
+- Kubernetes 1.9+ cluster
+- `docker` client
+- `kubectl` client
+- Helm Chart
+
+## Instructions
+
+1) Run the following:
+
+```sh
+$ git checkout git@github.com:operator-framework/helm-app-operator-kit.git && cd helm-app-operator-kit
+$ docker build -t quay.io/<namespace>/<chart>-operator -e HELM_CHART=/path/to/helm/chart -e API_GROUP=<group/version> -e KIND=<Kind> .
+$ docker push quay.io/<namespace>/<chart>-operator
+```
+
+2) Modify the following Kubernetes YAML manifest files in `helm-app-operator/deploy`:
+
+File                          | Action
+------------------------------|--------------------------------------------------------------------------------------------------------
+`deploy/crd.yaml`             | Define your CRD (`kind`, `spec.version`, `spec.group` *must* match the `docker build` args)
```

Comment:
Why do we need `deploy/crd.yaml` and `deploy/olm-catalog/crd.yaml`?

Reply:
The `operator-sdk build` command generates both, I think it's so you can `kubectl create -f `deploy/olm-catalog` and ignore the upper level files.